### ISSUE TITLE
question about loss function

### ISSUE THREAD

**Issue by nicetomeetu21** [ExplainableML/UncerGuidedI2I#4]
https://github.com/ExplainableML/UncerGuidedI2I/blob/cc4a4e2d0b84e97b072534b46337d7c7a50d18e0/src/losses.py#L79

Amazing work! But I still have some questions about the loss calculation. It seems different from the paper's formula. Why the Beta in the exponential of residual is replaced to multiply to the residual and the clamp operation would not break the formula?

**Comment by udion** [comment]
Hi,
You could use the exponential version as described in the paper, but I found it to be more sensitive on learning rate.

Multiplying and clamping is more robust and easily trainable.
